src/Extension/MemberExtension.php

```diff
@@ -71,9 +73,36 @@ public function updateCMSFields(FieldList $fields)
             $methodListField->setReadonly(true);
         }
 
+        // Don't show a read only field if the user cannot edit this record
+        // Don't show if a user views their own profile (just let them reset their own password)
+        if ($this->owner->canEdit() && !$this->currentUserIsThisMember()) {
+            $requireNewPassword = CheckboxField::create(
+                'RequirePasswordChangeOnNextLogin',
+                _t(static::class . 'RequirePasswordChangeOnNextLogin', 'Require password change on next login')
+            );
+            $fields->insertAfter('Password', $requireNewPassword);
+
+            $fields->fieldByName('Root.Main.Password')->addExtraClass('form-group--no-divider');
```

Comment:
You could use `$fields->dataFieldByName('Password')` to remove any ambiguity around which tab it would be placed in